examples/wasm/example_algorithms.json

```diff
@@ -1,12 +1,13 @@
-{
-    "algorithms": [
-        "alpha",
-        "multiplicative", {
-            "algorithm": "mask_top",
-            "params": {
-                "factor": 1.6,
-                "tobias": true
-            }
-        }
-    ]
-}
+{
+  "algorithms": [
```

Comment:
use cleanup to standardize naming and 4 spaced `pip install admin_scripts` then `cleanup .`